keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone_presets.py

```diff
@@ -0,0 +1,194 @@
+                0.25,
+            ],
+            "stackwise_activation": [
+                layers.ReLU(),
+                layers.ReLU(),
+                layers.ReLU(),
+                backbone.apply_hard_swish,
+                backbone.apply_hard_swish,
+                backbone.apply_hard_swish,
+                backbone.apply_hard_swish,
+                backbone.apply_hard_swish,
+                backbone.apply_hard_swish,
+                backbone.apply_hard_swish,
+                backbone.apply_hard_swish,
+            ],
+            "filters": 1024,
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "alpha": 1.0,
+            "dropout_rate": 0.2,
+        },
+    },
+    "mobilenetv3large": {
+        "metadata": {
+            "description": (
+                "MobileNetV3 model with 28 layers where the batch "
+                "normalization and hard-swish activation are applied after the "
+                "convolution layers."
+            ),
+        },
+        "class_name": "keras_cv.models>MobileNetV3Backbone",
+        "config": {
+            "stackwise_expansion": [
+                1,
+                4,
+                3,
+                3,
+                3,
+                3,
+                6,
+                2.5,
+                2.3,
+                2.3,
+                6,
+                6,
+                6,
+                6,
+                6,
+            ],
+            "stackwise_filters": [
+                16,
+                24,
+                24,
+                40,
+                40,
+                40,
+                80,
+                80,
+                80,
+                80,
+                112,
+                112,
+                160,
+                160,
+                160,
+            ],
+            "stackwise_kernel_size": [
+                3,
+                3,
+                3,
+                5,
+                5,
+                5,
+                3,
+                3,
+                3,
+                3,
+                3,
+                3,
+                5,
+                5,
+                5,
+            ],
+            "stackwise_stride": [1, 2, 1, 2, 1, 1, 2, 1, 1, 1, 1, 1, 2, 1, 1],
+            "stackwise_se_ratio": [
+                None,
+                None,
+                None,
+                0.25,
+                0.25,
+                0.25,
+                None,
+                None,
+                None,
+                None,
+                0.25,
+                0.25,
+                0.25,
+                0.25,
+                0.25,
+            ],
+            "stackwise_activation": [
+                layers.ReLU(),
+                layers.ReLU(),
+                layers.ReLU(),
+                layers.ReLU(),
+                layers.ReLU(),
+                layers.ReLU(),
+                backbone.apply_hard_swish,
```

Comment:
I don't think this is a property of the backbone (and shouldn't be!). Please check the import